### ISSUE TITLE
JS based scrolling through volumes

### ISSUE THREAD

**Issue by chrisfilo** [poldracklab/niworkflows#252]
Courtesy of @akeshavan http://anisha.pizza/preAFQ/#/report?url=https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fakeshavan-mindcontrol%2Ftests%2FHBN%2FSite-SI%2Fsub-NDARRU600XWW%2Fsess-01%2Fderivatives%2FpreAFQ%2Fpreafq%2Freport%2Freport.json

![screenshot_22](https://user-images.githubusercontent.com/238759/45369229-67cc0f00-b5b3-11e8-8905-af38cf70b14d.png)
![screenshot_23](https://user-images.githubusercontent.com/238759/45369230-68fd3c00-b5b3-11e8-9e67-b56653cda34d.png)

**Comment by oesteban** [comment]
I really wanted to have this in MRIQC! 👍

**Comment by chrisfilo** [comment]
Yeah looks sorry cool and very responsive (used sprites). I hear the code is not very complex so hopefully it will not be a lot of work.

**Comment by akeshavan** [comment]
Relevant part of code is this: https://github.com/akeshavan/preAFQ/blob/master/preafqViewer/src/components/Sprite4D.vue#L18

Needs to be cleaned up a bit, but if you have a sprite w/ `M` columns, and each sprite image is `pix` in each direction (i force my sprites to be square) then you can calculate the starting pixels in the X and Y direction as a function of `time`, and you set the CSS accordingly.
